ui/pages/send/send.test.js

```diff
@@ -0,0 +1,173 @@
+          expect.objectContaining({
+            type: 'send/initializeSendState/pending',
+          }),
+          expect.objectContaining({
+            type: 'metamask/gas/BASIC_GAS_ESTIMATE_STATUS',
+          }),
+          expect.objectContaining({
+            type: 'metamask/gas/SET_ESTIMATE_SOURCE',
+          }),
+        ]),
+      );
+    });
+
+    it('should showQrScanner when location.search is ?scan=true', () => {
+      useLocation.mockImplementation(() => ({ search: '?scan=true' }));
+      const store = configureMockStore(middleware)(baseStore);
+      renderWithProvider(<Send />, store);
+      const actions = store.getActions();
+      expect(actions).toStrictEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
+            type: 'ENS/enableEnsLookup',
+          }),
+          expect.objectContaining({
+            type: 'send/initializeSendState/pending',
+          }),
+          expect.objectContaining({
+            type: 'metamask/gas/BASIC_GAS_ESTIMATE_STATUS',
+          }),
+          expect.objectContaining({
+            type: 'metamask/gas/SET_ESTIMATE_SOURCE',
+          }),
+          expect.objectContaining({
+            type: 'UI_MODAL_OPEN',
+            payload: { name: 'QR_SCANNER' },
+          }),
+        ]),
+      );
+      useLocation.mockImplementation(() => ({ search: '' }));
+    });
+  });
+
+  describe('Add Recipient Flow', () => {
+    it('should render the header with Add Recipient displayed', () => {
+      const store = configureMockStore(middleware)(baseStore);
+      const { getByText } = renderWithProvider(<Send />, store);
+      expect(getByText('Add Recipient')).toBeTruthy();
+    });
+
+    it('should render the EnsInput field', () => {
+      const store = configureMockStore(middleware)(baseStore);
+      const { getByPlaceholderText } = renderWithProvider(<Send />, store);
+      expect(
+        getByPlaceholderText('Search, public address (0x), or ENS'),
+      ).toBeTruthy();
+    });
+
+    it('should not render the footer', () => {
+      const store = configureMockStore(middleware)(baseStore);
+      const { queryByText } = renderWithProvider(<Send />, store);
+      expect(queryByText('Next')).toBeNull();
+    });
+  });
+
+  describe('Send and Edit Flow', () => {
+    it('should render the header with Send displayed', () => {
+      const store = configureMockStore(middleware)({
+        ...baseStore,
+        send: { ...baseStore.send, stage: SEND_STAGES.DRAFT },
+      });
+      const { getByText } = renderWithProvider(<Send />, store);
+      expect(getByText('Send')).toBeTruthy();
+    });
+
+    it('should render the EnsInput field', () => {
+      const store = configureMockStore(middleware)(baseStore);
+      const { getByPlaceholderText } = renderWithProvider(<Send />, store);
+      expect(
+        getByPlaceholderText('Search, public address (0x), or ENS'),
+      ).toBeTruthy();
+    });
+
+    it('should  render the footer', () => {
```

Comment:
sorry was in the middle of all the other changes so just made this change instead of accepting suggestion :( https://github.com/MetaMask/metamask-extension/pull/10965/commits/8ef1d5eed46a82125aaa66c75a463df100c69506